samples/openthread/cli/README.rst

```diff
@@ -339,6 +339,113 @@ To test the Thread 1.2 features, complete the following steps:
    .. note::
         The DUA registration with the Backbone Router is not yet supported.
 
+#. Link Metrics
+
+   * Perform a Link Metrics query (Single Probe).
```

Comment:
We should indicate in which device to apply the commands. For this purpose, we can modify the configuration above to have a SED instead of a Router, apart from the Leader.